client/src/leap/soledad/client/interfaces.py

```diff
@@ -20,6 +20,18 @@
 from zope.interface import Interface, Attribute
 
 
+class ISoledadSyncedPlugin(Interface):
+
+    def process_received_docs(self, doc_id_list):
+        """
+        Do something with the passed list of doc_ids received after the last
+        sync.
+        """
+
+    watched_doc_types = Attribute("""
+        a tuple of the watched doc types for this plugin""")
```

Comment:
you mean if the current implementation of the interface is actually implementing the attribute?
if it's that, it does:
https://github.com/kalikaneko/leap_mail/commit/ee4bb4117c202770df7615f2623032b2021b2a65#diff-0ddf4e813820f3c8cb170b906d84a1baR35

if you meant otherwise, can you elaborate?